internal/ethapi/testdata/transfer_after_transfer.json

```diff
@@ -0,0 +1,3 @@
+{
+
```

Comment:
Empty json file? Should we delete it?

Reply:
Ah must have been something I wanted to do but changed my mind about. Removed thank you.